### DISCUSSION TITLE
Linear Main chain

### DISCUSSION THREAD

**Discussion by percyzt** [wevm/wagmi#2758] [Q&A]
When did the Linea main network start supporting it? It was not found in the provided chain program. May I know when the Linea main network was supported?

**Comment by minhhoang2404** [comment]
you can config it same as example : 
export const mantle = {
  id: 5001,
  name: 'Mantle Testnet',
  network: 'mantle',
  iconUrl: 'https://i.imgur.com/Q3oIdip.png',
  iconBackground: '#000000',
  nativeCurrency: {
    decimals: 18,
    name: 'BIT',
    symbol: 'BIT',
  },
  rpcUrls: {
    default: {
      http: ['https://rpc.testnet.mantle.xyz'],
    },
    public: {
      http: ['https://rpc.testnet.mantle.xyz'],
      // public rpc url
    },
  },
  blockExplorers: {
    default: {
      name: 'Mantle Testnet Explorer',
      url: 'https://explorer.testnet.mantle.xyz',
    },
    etherscan: {
      name: 'Mantle Testnet Explorer',
      url: 'https://explorer.testnet.mantle.xyz',
    },
  },
  testnet: true,
};
